src/state/reducer.ts

```diff
@@ -19,6 +19,28 @@ function isMergeable<T>(constructor: any): constructor is MergeableStatic<T> {
   return constructor && typeof constructor.merge === 'function';
 }
 
+function purgeEntity(
```

Comment:
why?

Reply:
why not? Makes reducer more readable and better function abstraction IMO. Why did I change it here? Because I was originally thinking I was going to re-use it. I can make this a separate PR if you want.

Reply:
meh, I don't think it is much better either way, but I would use the type directly instead of init:

State<unknown>['entities']